### DISCUSSION TITLE
hci_usb sample Data Length Extension related commands not supported

### DISCUSSION THREAD

**Discussion by abferm** [zephyrproject-rtos/zephyr#33585] [General] [answered]
I am trying to use the nRF52840-DK as a USB HCI device, but am getting Unknown HCI command responses to "LE Read Maximum Data Length", "LE Read Suggested Default Data Length", and "LE Write Suggested Default Data Length".

I found [this](https://github.com/zephyrproject-rtos/zephyr/blob/2affd52339250733285f8e52602c312364901289/subsys/bluetooth/controller/hci/hci.c#L749) but am not sure if enabling this option will solve my issue, or how to go about enabling it.

**Comment by carlescufi** [comment, marked as answer]
Yes, you do need to enable that option. Add `CONFIG_BT_CTLR_DATA_LENGTH=y` in your `prj.conf`
